client/state/jetpack-onboarding/reducer.js

```diff
@@ -0,0 +1,21 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { createReducer, combineReducers, keyedReducer } from 'state/utils';
+import { JETPACK_ONBOARDING_CREDENTIALS_RECEIVE } from 'state/action-types';
+
+export const credentialsReducer = keyedReducer(
+	'siteId',
+	createReducer(
+		{},
+		{
+			[ JETPACK_ONBOARDING_CREDENTIALS_RECEIVE ]: ( state, { credentials } ) => credentials,
+		}
+	)
+);
+
+export default combineReducers( {
```

Comment:
Are we expecting more than one subreducer? If we aren't, should be fine to drop the `combineReducer()` (YAGNI and all -- as always, I'd personally actually start without `combineReducer` anyway and only add it once needed 😄 ).

Reply:
Well, I think we'll have some more subreducers here 😉  Happy to remove the `combineReducer` if you disagree though.

Reply:
Depends a bit on which data tooling strategy we'll eventually settle I guess. For now, your call again 😁